packages/dev/core/src/Shaders/ShadersInclude/pbrDebug.fx

```diff
@@ -128,6 +128,14 @@ if (vClipSpacePosition.x / vClipSpacePosition.w >= vDebugMode.x) {
         gl_FragColor.rgb = subSurfaceOut.transmittance;
     #elif DEBUGMODE == 70 && defined(SUBSURFACE) && defined(SS_REFRACTION)
         gl_FragColor.rgb = subSurfaceOut.refractionTransmittance;
+    #elif DEBUGMODE == 72
+        gl_FragColor.rgb = vec3(1.0-roughness); // TODO
+    #elif DEBUGMODE == 73
+        gl_FragColor.rgb = vAlbedoColor.rgb; // TODO

```

Comment:
vAlbedoColor is the material color, are you sure it is the one you are looking to display ? (looks like it but just to be sure)

There are up to 5 albedos on a PBR material, the vertex color (from the attributes data), the instance color (in case of instancing each instance can have a different color), the material base color (from the material data) and the texture one, the detail albedo from a detail texture.

The runtime is using a product of the 5 if all defined.

We need to be clear which one we want to troubleshoot :-)

Also this would benefit from the `#define DEBUGMODE_GAMMA` cause the albedo color is in linear space.